percona/version/version.go

```diff
@@ -0,0 +1,95 @@
+package version
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/percona/percona-postgresql-operator/internal/logging"
+	"github.com/percona/percona-postgresql-operator/percona/version/service/client"
+	"github.com/percona/percona-postgresql-operator/percona/version/service/client/version_service"
+	"github.com/percona/percona-postgresql-operator/pkg/apis/pg.percona.com/v2beta1"
+)
+
+const (
+	Version     = "2.0.0"
+	productName = "pg-operator"
+)
+
+type DepVersion struct{}
+
+type Meta struct {
+	Apply           string
+	OperatorVersion string
+	PGVersion       string
+	KubeVersion     string
+	Platform        string
+	PMMVersion      string
+	BackupVersion   string
+	CRUID           string
+}
+
+func EnsureVersion(ctx context.Context, cr *v2beta1.PerconaPGCluster, meta Meta) error {
+	log := logging.FromContext(ctx)
+
+	if !telemetryEnabled() {
+		return nil
+	}
+
+	_, err := fetchVersions(ctx, cr, v2beta1.GetDefaultVersionServiceEndpoint(), meta)
+	if err != nil {
+		// we don't return here to not block execution just because we can't phone home
+		log.Error(err, fmt.Sprintf("failed to send telemetry to %s", v2beta1.GetDefaultVersionServiceEndpoint()))
+	}
+
+	return nil
+}
+
+func telemetryEnabled() bool {
+	value, ok := os.LookupEnv("DISABLE_TELEMETRY")
```

Comment:
please add it to operator/bundle yaml

Reply:
https://github.com/percona/percona-postgresql-operator/pull/314/commits/d1069a85b8a83139a89ac1b830e6a9e1a201298b